Main.java

```diff
@@ -9,7 +10,32 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    Scanner input = new Scanner(System.in);
+
+    final int num1;
+
+    final int num2;
+
+  System.out.println("Please enter a integer");
+  int integer= input.nextInt ();
+
+  double quotient =1 / 2;
+  System.out.println("that is a even number");
+
```

Comment:
For this question, we need to check the remainder. So we can do the modulus when we divide by 2 and store it in a variable. Once we have that, all we need to do is check to see if it is equal to 1 or 0. 0 means it is even, 1 means it is odd.